release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+
+{product-title} leverages Transport Layer Security (TLS) cipher suites, JSON Web
+Algorithms (JWA) crypto algorithms, and offers external libraries such as The
+Generic Security Service Application Program Interface (GSSAPI) and libgpgme.
+
+xref:../architecture/index.adoc#architecture-index[Private and public key
+configurations and Crypto levels] are now documented for {product-title}.
+
+[[ocp-37-node-authorizer-node-restriction-admission-plug-in]]
+==== Node Authorizer and Node Restriction Admission Plug-in
+
+Pods can no longer try to gain information from secrets, configuration maps, PV,
+PVC, or API objects from other nodes.
+
+link:https://kubernetes.io/docs/admin/authorization/node/[Node authorizer]
+governs what APIs a kublet can perform. Spanning read-, write-, and auth-related
+operations. In order for the admission controller to know the identity of the
+node to enforce the rules, nodes are provisioned with credentials that identify
+them with the user name `system:node:<nodename>` and group `system:nodes`.
+
+These enforcements are in place by default on all new installations of
+{product-title} 3.7.  For upgrades from {product-title} 3.6, they are not in
+place due to the `system:nodes` RBAC being granted from OCP 3.6. To turn the
+enforcements on, run:
+
+----
+# oadm policy remove-cluster-role-from-group system:node system:nodes
+----
+
+[[ocp-37-advanced-auditing]]
+==== Advanced Auditing (Technology Preview)
+
+With Advanced Auditing (currently in xref:ocp-37-technology-preview[Technology
+Preview]), administrators are now exposed to more information from the API call
+within the audit trail. This provides a deeper traceability of what is occurring
+across the cluster.  We also capture all login events at the default logging
+level and modifications to role binds and SCC.
+
+{product-title} now has an audit `policyFile` or `policyConfiguration` where
+administrators can filter in on what they want to capture.
+
+See
+xref:../install_config/master_node_configuration.adoc#master-node-config-advanced-audit[Advanced
+Audit] for more information.
+
+[[ocp-37-complete-upstreaming-of-rbac-then-downstreaming]]
+==== Complete Upstreaming of RBAC, Then Downstreaming it Back into OpenShift
+
+The rolebinding and RBAC experience is now the same across all Kubernetes
+distributions.
+
+Administrators do not have to do anything for this migration to occur. The
+upgrade process to {product-title} 3.7 offers a seamless experience. Now, the
+user experience is consistent with upstream.
+
+A role can be defined within a namespace with a `Role`, or cluster-wide with a
+`ClusterRole`.
+
+A `RoleBinding` or `ClusterRoleBinding` binds a role to subjects. Subjects can
+be groups, users, or service accounts. A role binding grants the permissions
+defined in a role.
+
+[[ocp-37-flexvolume-support-for-non-stotage-use-cases]]
+==== Official FlexVolune Support for Non-storage Use Cases
```

Comment:
This is more like a storage feature, but for the security part, maybe we can write something about "SCC now support flexVolume" that we did in this card: https://trello.com/c/YT6sNEay/61-5-scc-flex-volume-support-sccfsi37

Reply:
BTW, typo in "FlexVloune"